src/content/docs/tutorial-dd-migration/installing-agent.mdx

```diff
@@ -0,0 +1,448 @@
+<TabsPageItem id="3">
+<SideBySide>
+    <Side>
+        <img
+            title="log-dashboard"
+            alt="An image displaying New Relic's log monitoring dashboard"
+            src={journeyLogsDash}
+        />
+    </Side>
+    <Side>
+Log observability is essential to both understanding and using your data to the fullest by giving you information on how your users interact with your applications. We believe log observability should be easy to manage, easy to access, and easy to scale. New Relic lets you ingest any text-based data from any source, letting you easily manage large quantities of your data in one place. 
+
+Once you've set your logs up, you can even utilize machine learning to help you find patterns and anomalies fast without the need for manual querying. Like infrastructure and alert agents, we offer a host of guided installation options as well as preconfigured dashboards and alerts. And also like infrastructure and application agents, you can try out logging [for free](https://newrelic.com/signup).
+    </Side>
+</SideBySide>
+
+## Your log monitoring solution [#logs-solution]
+
+Logs in the New Relic UI are found under the **Logs** section. The hard part with log data is usually figuring out what matters. But it's also important to connect what's happening in your logs with the rest of your system. So we surface relevant logs in APM, infrastructure, and elsewhere to help you troubleshoot.
+
+You can automatically pinpoint events, errors, traces, and spans among your connected entities with just a single click, keeping you in one place and getting you to what you're looking for that much faster. Our log monitoring solution:
+
+* Enables deep visibility into your entity data, letting you see how your system works together.
+* Helps reduce your mean time to resolution (MTTR), letting you respond to issues faster.
+* Gives you the ability to create curated charts, dashboards, and alerts, giving you as detailed a look into your data as you want.
+* Provides the ability to visualize, search, and troubleshoot performance issues in one central location without the need to switch between tools
+## Get started with New Relic log monitoring [#get-started].
+
+Once installed, your log monitoring is done in the **Logs** section of the UI, exactly like it is in Datadog. You can use our guided installations for logs after setting up any of our [infrastructure](/docs/journey-migration/migrating-infra/) or [APM](/docs/journey-migration/migrating-apm/) agents. 
+
+The steps for setting up logs varies depending on the data source, but no matter which you choose, you can rest assured knowing we'll guide you through every step of the process to make it as easy and straightforward as possible. You can see our list of guided installations for logs by selecting **one.newrelic.com > Add Data > Logging** in the New Relic UI.
+
+        </TabsPageItem>
+    </TabsPages>
+</Tabs>
+
+## Explore premade dashboards [#add-dashboard]
```

Comment:
I'm not sure why but in the preview, this header looks weird and flush with the preceding paragraph